MacaoProject/Macao/Card A.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
filenames should not have spaces + i said to put all the models in a separate folder